pkgs/applications/science/electronics/linux-gpib/linux-gpib-user/default.nix

```diff
@@ -0,0 +1,46 @@
+{ lib
+, stdenv
+, fetchurl
+, autoconf
+, libtool
+, bison
+, flex
+, automake
+}:
+
+stdenv.mkDerivation rec {
+  pname = "linux-gpib-user";
+  version = "4.3.6";
+
+  src = fetchurl {
+    url = "https://sourceforge.net/projects/linux-gpib/files/linux-gpib%20for%203.x.x%20and%202.6.x%20kernels/${version}/linux-gpib-${version}.tar.gz";
+    hash = "sha256-Gze4xrvkhEgn+J5Jhrycezjp2uhlD1v6aX0WGv4J2Jg=";
+  };
+
+  unpackPhase = ''
+    tar xf $src
+    tar xf linux-gpib-${version}/${pname}-${version}.tar.gz
+  '';
+
+  sourceRoot = "${pname}-${version}";
+
+  nativeBuildInputs = [
+    autoconf
+    libtool
+    bison
+    flex
+    automake
+  ];
+
+  configurePhase = ''
+    ./configure --sysconfdir=$out/etc --prefix=$out
+  '';
+
+  meta = with lib; {
+    description = "Support package for GPIB (IEEE 488) hardware";
+    homepage = "https://linux-gpib.sourceforge.io/";
+    license = licenses.gpl3;
```

Comment:
is that gpl3Only or gpl3Plus?

Reply:
Updated to `gpl2Only`, for more consistency to the package documentation.